VinylUI.Tests/WinFormsTests.fs

```diff
@@ -186,6 +186,23 @@ let ``Bind nullable to option two-way for custom control`` () =
     binding |> testModelToView viewExpr (Option.toNullable model.Age) (Some 31) (Nullable 31)
     binding |> testViewInpcToModel viewExpr model.Age (Nullable 32) (Some 32)
 
+[<TestCaseSource("sourceUpdateModes")>]
+let ``Bind string to string option two-way`` sourceUpdate =
+    use form = new FakeForm()
+    let viewExpr = <@ form.TextBox.Text @>
+    let binding = Bind.view(viewExpr).toModel(<@ model.NickName@>, sourceUpdate)
+    binding |> testModelToView viewExpr (Option.defaultValue "" model.NickName) (Some "D") ("D")
+    binding |> testViewToModel sourceUpdate viewExpr model.NickName ("D") (Some "D")
+
+
+[<Test>]
+let ``Bind string to string option two-way for custom control`` () =
+    use form = new FakeForm()
+    let viewExpr = <@ form.CustomTextControl.Value @>
+    let binding = Bind.viewInpc(viewExpr).toModel(<@ model.NickName @>)
+    binding |> testModelToView viewExpr (Option.defaultValue "" model.NickName) (Some "D") ("D")
+    binding |> testViewInpcToModel viewExpr model.NickName ("D") (Some "D")
+
```

Comment:
It doesn't look like the tests cover null, empty and whitespace inputs, which are the main cases to cover.